bestiary/core/models.py

```diff
@@ -202,6 +240,8 @@ class Ecosystem(EntityBase):
     description = CharField(max_length=MAX_SIZE_CHAR_FIELD,
                             null=True,
                             help_text='Brief text describing the ecosystem')
+    projects = ManyToManyField(Project,
```

Comment:
I see this interesting but I'm not sure if it creates more problems and adds complexity to the system. The good thing is we can have the same project in different ecosystems, so ecosystems are synchronized. The main problem is what happens when the user wants to have almost the same project structure but not equal.

Reply:
I see your point. As we began to define in #39, a project is just a set of existing end-point views, so if the user wants to have a very similar project structure, the project can be copied selecting which end-point views remain, which ones don't and which ones are new. WDYT? Is this something you want to avoid? Thanks!

Reply:
Not sure. I understand the point about the views but I don't think projects structure should be a fixed one. Once you define one and you have it in two ecosystems, that means any change you make in one ecosystem is changed in the second one too.

Reply:
You are right, I think this adds unnecessary complexity to the system. I changed the definition of the link between Ecosystems and Projects to 1:n.